### PR TITLE
Adds fixes to defaults that should have been there from the beginning

### PR DESCRIPTION
**Pull request by Scriptkiddi**
Adds quoting for some default vars because yaml was parsing those to boolean values
Adds checks for 2 variables that are not defined in the defaults

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,9 +1,5 @@
 ---
-- name: Update Cache
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: Why did you remove this task?

File: templates/redis.conf.j2 [hunk 2]
```diff
@@ -36,10 +36,12 @@
 #
 # include /path/to/local.conf
 # include /path/to/other.conf
-
+{% if redis_include is defined %}
 {% for redis_include_opt in redis_include %}
```
Comment (SlothOfAnarchy) [inline templates/redis.conf.j2]: why not use `|default([])`?

File: templates/redis.conf.j2 [hunk 3]
```diff
@@ -525,9 +527,12 @@ requirepass {{ redis_requirepass }}
 #
 # rename-command CONFIG ""
 #
+{% if redis_rename_command is defined %}
 {% for redis_rename_command_opt in redis_rename_command %}
```
Comment (SlothOfAnarchy) [inline templates/redis.conf.j2]: same here

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy  pls rereview
